src/nginxconfig/templates/domain_sections/php.vue

```diff
@@ -65,6 +65,54 @@ THE SOFTWARE.
             </div>
         </div>
 
+        <div v-if="phpServerEnabled" class="field is-horizontal">
+            <div class="field-label has-margin-top">
+                <label class="label">{{ $t('templates.domainSections.php.phpServer') }}</label>
+            </div>
+            <div class="field-body">
+                <div class="field">
+                    <div :class="`control${phpServerChanged ? ' is-changed' : ''}`">
+                        <VueSelect ref="phpServerSelect"
+                                   v-model="phpServer"
+                                   :options="phpServerOptions"
+                                   :clearable="false"
+                                   :reduce="s => s.value"
+                        ></VueSelect>
+                    </div>
+
+                    <div v-if="phpServerCustomEnabled"
+                         :class="`control${phpServerCustomChanged ? ' is-changed' : ''}`"
+                    >
+                        <input v-model="phpServerCustom" class="input" type="text" :placeholder="$props.data.phpServerCustom.default" />
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div v-if="phpBackupServerEnabled" class="field is-horizontal">
```

Comment:
```suggestion
        <div v-if="phpBackupServerEnabled" class="field is-horizontal is-aligned-top">
```
Looks like we lost the `is-aligned-top` class here.